scss/tech.scss

```diff
@@ -0,0 +1,34 @@
+
+.border{
+width: 200px;
```

Comment:
Check indentation, this should have two spaces at the left, the same with all elements contained